pegasus/sites.v3/code.org/public/index.haml

```diff
@@ -19,8 +19,12 @@ style_min: true
 -# Apply this stylesheet when 'pl-launch-hero-banner' is set to 'true'
 - if !!DCDO.get('pl-launch-hero-banner', false)
   =inline_css '/generated/pl-hero-banner.css'
+-# Apply this stylesheet when 'curriculum-launch-2024' is set to 'true'
+- if !!DCDO.get('curriculum-launch-2024', false)
+  =inline_css '/generated/banners-curriculum-launch.css'
 -# TODO: Remove this after the Music Lab launch is over
-=inline_css '/generated/music-lab-hero-banner.css'
+- if !!DCDO.get('music-lab-launch-2024', false)
+  =inline_css '/generated/music-lab-hero-banner.css'
```

Comment:
Should there be an extra line of logic here where one banner takes precedence if both flags are true so that we don't end up in a state where the banners are back to back? Or is that okay?

Reply:
If both flags are set to true the Music Lab banner will take precedence since it's first in order [here](https://github.com/code-dot-org/code-dot-org/blob/38f42be85693508bdaa6c3eabf2421fe44e2e37b/pegasus/sites.v3/code.org/homepage.json#L2-L7) so they won't show at the same time.

We'll just need to turn off the `music-lab-launch-2024` DCDO flag before flipping the new `curriculum-launch-2024` flag.

Reply:
I also made a [ticket](https://codedotorg.atlassian.net/browse/ACQ-1981) to deprecate the Music Lab launch stuff at some point in the future. cc @bethanyaconnor and @dmcavoy

Reply:
Oh right forgot that it handles the ordering! Thank you!